librz/core/visual.c

```diff
@@ -2771,25 +2778,21 @@ RZ_API int rz_core_visual_cmd(RzCore *core, const char *arg) {
 			visual_refresh(core);
 			break;
 		case ' ':
-		case 'V':
-			if (rz_config_get_b(core->config, "graph.web")) {
```

Comment:
I removed this case since it doesn't work either and essentially shows also an ASCII graph too.

Reply:
I'd kill graph.web completely or open an issue to do it in another PR.

Reply:
> I'd kill graph.web completely or open an issue to do it in another PR.


I'll look into that.

Reply:
Right